### ISSUE TITLE
How does the AutoML markdown report calculate metrics?

### ISSUE THREAD

**Issue by prateekchandrajha** [mljar/mljar-supervised#484]
The automl.report() churns out a KS curve which gives a KS of 0.564 at 0.167.

When I take the automl object and draw the decile tables on the training data, it gives me the KS of 90.052 at decile 2.

The test data KS is around 50.605 at decile 4.

Can you in detail let us know the exact way in which metrics are calculated for the best automl model in the markdown report?

Regards & Thanks

Prateek.

**Comment by pplonski** [comment]
Hi @prateekchandrajha, MLJAR AutoML is using validation data (out-of-sample predictions) to compute KS. What validation or mode are you using? If Explain mode with 80/20 split then only 20% of data (validation set) will be used to compute KS. If you use cross-validation, then out-of-sample predictions are collected from CV and they are used to compute KS.
